Fluid.MvcViewEngine/Fluid.MvcViewEngine.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework>netstandard2.1</TargetFramework>
```

Comment:
can this be a problem for people upgrading? no longer usable on full framework

Reply:
Parlot requirement. ReadOnlySpan ... So if it's fixed it has to be in Parlot